static/app/components/nav/primary.tsx

```diff
@@ -240,31 +258,38 @@ export function PrimaryNavigationItems() {
   );
 }
 
-const SidebarItemList = styled('ul')`
+const SidebarItemList = styled('ul')<{isMobile: boolean}>`
   position: relative;
   list-style: none;
   margin: 0;
   padding: 0;
   padding-top: ${space(1)};
   display: flex;
   flex-direction: column;
+  align-items: stretch;
+  gap: ${space(0.5)};
   width: 100%;
   color: rgba(255, 255, 255, 0.85);
 
-  @media screen and (min-width: ${p => p.theme.breakpoints.medium}) {
-    gap: ${space(1)};
-  }
+  ${p =>
+    !p.isMobile &&
+    css`
+      align-items: center;
+      gap: ${space(1)};
+    `}
 `;
 
-const SidebarItemWrapper = styled('li')`
+const SidebarItemWrapper = styled('li')<{isMobile: boolean}>`
```

Comment:
feels like you could do something to make isMobile css only here if you wanted

Reply:
Yeah we actually had more CSS media queries before, but since we need `isMobile` in the context anyway I figured it would be cleaner to just use the JS completely. People won't be switching back and forth often so I figure it's probably a fine tradeoff